go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/smartcontractkit/chainlink-common
 
-go 1.22.0
+go 1.23
 
-toolchain go1.22.7
```

Comment:
This and the other changes to go.mod are all from `go get github.com/marcboeker/go-duckdb`. It requires go 1.23